rdr_service/alembic/versions/afb0333cb471_adding_ignore_field_to_questionnaire_.py

```diff
@@ -0,0 +1,62 @@
+"""adding ignore field to questionnaire response answer
+
+Revision ID: afb0333cb471
+Revises: f97587ebf104
+Create Date: 2022-01-06 11:20:44.446748
+
+"""
+from alembic import op
+import sqlalchemy as sa
+import rdr_service.model.utils
+
+
+from rdr_service.participant_enums import PhysicalMeasurementsStatus, QuestionnaireStatus, OrderStatus
+from rdr_service.participant_enums import WithdrawalStatus, WithdrawalReason, SuspensionStatus, QuestionnaireDefinitionStatus
+from rdr_service.participant_enums import EnrollmentStatus, Race, SampleStatus, OrganizationType, BiobankOrderStatus
+from rdr_service.participant_enums import OrderShipmentTrackingStatus, OrderShipmentStatus
+from rdr_service.participant_enums import MetricSetType, MetricsKey, GenderIdentity
+from rdr_service.model.base import add_table_history_table, drop_table_history_table
+from rdr_service.model.code import CodeType
+from rdr_service.model.site_enums import SiteStatus, EnrollingStatus, DigitalSchedulingStatus, ObsoleteStatus
+
+# revision identifiers, used by Alembic.
+revision = 'afb0333cb471'
+down_revision = 'f97587ebf104'
+branch_labels = None
+depends_on = None
+
+
+def upgrade(engine_name):
+    globals()["upgrade_%s" % engine_name]()
+
+
+def downgrade(engine_name):
+    globals()["downgrade_%s" % engine_name]()
+
+
+
+def upgrade_rdr():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('questionnaire_response_answer', sa.Column('ignore', sa.Boolean(), nullable=True))
```

Comment:
Just FYI, `ignore` is a reserved word in MySQL. It should be fine here, but you'll need to wrap it in back-ticks if you use raw SQL.

Reply:
Ah, ok. Thanks. I'm sure escaping it will be fine if we need to reference it in raw sql.